coalib/bears/requirements/PackageRequirement.py

```diff
@@ -49,3 +49,73 @@ def check(self):
         :return: Returns True if satisfied, False if not.
         """
         raise NotImplementedError
+
+    @classmethod
+    def multiple(cls, *args):
+        """
+        Creates a tuple of multiple instances of a class.
+
+        Should not be instances of ``PackageRequirement``, as this is an
+        abstract class:
+
+        >>> PackageRequirement.multiple(('pip', 'coala_decorators', '0.1.0'),)
+        Traceback (most recent call last):
+        ...
+        NotImplementedError
+
+        It can only be used for requirements of the same manager. For example,
+        consider a manager ``XYZRequirement`` that inherits from
+        PackageRequirement. This subclass will have the manager set to XYZ:
+
+        >>> class XYZRequirement(PackageRequirement):
+        ...     manager = 'xyz'
```

Comment:
I think this manager variable doesn't need to be exposed to the user, can we make it `_manager` at least? Also it's a constant so it should rather be `_MANAGER` I think. This doesn't belong into this PR but maybe file an issue?